packages/cfpb-atomic-component/src/utilities/atomic-helpers.js

```diff
@@ -102,10 +102,11 @@ function setInitFlag(element) {
  * @param {string} selector - Selector to search for in the document.
  * @param {Function} Constructor - A constructor function.
  * @param {HTMLElement} [scope] - A dom node in which to query the selector.
+ * @param {object} config - Configuration will be provided to the Constructor's init()
  *   If not supplied, it defaults to the `document`.
  * @returns {Array} List of instances that were instantiated.
  */
-function instantiateAll(selector, Constructor, scope) {
+function instantiateAll(selector, Constructor, scope, config = {}) {
```

Comment:
You'll want to update the associated unit tests for this change.

Reply:
Added unit tests for configuration support to both Multiselect and instantiateAll